Benchmarks/Benchmarks/NIOPosixBenchmarks/Benchmarks.swift

```diff
@@ -64,4 +65,43 @@ let benchmarks = {
         )
     }
     #endif
+
+    Benchmark(
+        "MTELG.scheduleTask(in:_:)",
+        configuration: Benchmark.Configuration(
+            metrics: [.mallocCountTotal, .instructions],
+            scalingFactor: .kilo
```

Comment:
Can we align the maximum duration/iterations with https://github.com/apple/swift-nio/pull/2839